crates/subspace-farmer/src/bin/subspace-farmer/commands/farm.rs

```diff
@@ -103,13 +124,43 @@ async fn configure_dsn(
     enable_dsn: bool,
     listen_on: Vec<Multiaddr>,
     bootstrap_nodes: Vec<Multiaddr>,
-) -> Result<(Option<Node>, Option<NodeRunner>), anyhow::Error> {
+    record_cache_size: usize,
+    record_cache_db_path: Option<String>,
```

Comment:
I don't think we need a separate parameter for this. We should probably use `base_path` we already have and that can be used with temporary directory. It shouldn't magically switch to temporary directory either and if we even end up with separate path it shouldn't be a string, it should be a `Path` or `PathBuf`.

With that in mind, we should probably remove default value from `--base-path` CLI argument and change its description to something more generic (used for plots unless separate per-disk plots are provided too).